xarray/tests/test_plot.py

```diff
@@ -1806,3 +1805,23 @@ def test_yticks_kwarg(self, da):
         da.plot(yticks=np.arange(5))
         expected = np.arange(5)
         assert np.all(plt.gca().get_yticks() == expected)
+
+
+def test_multidim_coords_line_plot():
```

Comment:
This test should be added to a Unittest child class -> where are the other line plots tested?

Reply:
Oh it needed a `@requires_matplotlib`. I moved it up but I thought we were supposed to be transitioning to a classless testing code.